app/build.gradle

```diff
@@ -337,14 +337,24 @@ android.applicationVariants.all { variant ->
                 buildConfigField 'String', 'ADJUST_ENVIRONMENT', 'null'
             }
             println "Added adjust token set from environment variable"
+
+            def tracker = System.getenv("ADJUST_SIDELOAD_TRACKER") ?: null
+            if (tracker != null) {
+                buildConfigField 'String', 'ADJUST_DEFAULT_TRACKER', '"' + tracker + '"'
+            } else {
+                buildConfigField 'String', 'ADJUST_DEFAULT_TRACKER', 'null'
```

Comment:
Are we requiring all build with adjust token to add DEFAULT_TRACKER?

Reply:
all build with adjust token WITH ```ADJUST_SIDELOAD_TRACKER``` defined in environment variable will add DEFAULT_TRACKER